Eliminate a compiler warning under FreeBSD.

===================================================================
RCS file: /home/jdp/m3-cvs/m3/quake/src/utils.c,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 utils.c
--- m3/quake/src/utils.c.orig	1996/09/24 05:22:00	1.1.1.1
+++ m3/quake/src/utils.c	1996/09/24 05:32:34
@@ -10,7 +10,7 @@
  */
 
 #include "quake.h"
-#if !defined(TARGET_NEXT)
+#if !defined(TARGET_NEXT) && !defined(TARGET_FreeBSD2)
 #include <malloc.h>
 #endif
 
